content/applications/inventory_and_mrp/quality.rst

```diff
@@ -0,0 +1,20 @@
+:nosearch:
+:show-content:
+:hide-page-toc:
+:show-toc:
+
+=======
+Quality
+=======
+
+**Odoo Quality** helps ensure product quality throughout manufacturing processes and inventory
+movements. Conduct quality checks, automate quality inspection frequency, and create quality alerts
+when issues arise.
+
+.. seealso::
+   - `Odoo Tutorials: Managing Quality Checks <https://www.odoo.com/slides/slide/managing-quality-checks-708>`_
+
+.. toctree::
+   :titlesonly:
+
+   quality/quality_management
```

Comment:
I think this category is useless, and you could list `quality_control_points` and `quality_alerts` directly here, wdyt?

Reply:
I will be fleshing out the Quality scope with more docs over the coming months. They will need to be split across multiple sub-folders. I've already created the "Quality management" sub-folder for the control points and alerts docs in anticipation of that. I could remove the sub-folder for now, but it's going to be added back in shortly.